assets/templates/atoms/head.html

```diff
@@ -35,7 +35,20 @@
     <script defer src="{% static 'js/sb-admin-2.min.js' %}"></script>
     <script defer src="{% static 'js/project.min.js' %}"></script>
     {% endcompress %}
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-WW2W29ZMTZ"></script>
+
     <script src="https://kit.fontawesome.com/e1c9d8c99d.js" crossorigin="anonymous"></script>
     {% endblock javascript %}
+
+
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-S5JLWM36RQ"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
```

Comment:
I think we should strive to keep very little `javascript` embeded in `html` if not remove it entirely. This way, it easier to manage the `javascript` and might make it easier for us to switch to `Typescript` in the future.
I would recommend you move this to the `project.js` file.

Reply:
Agreed.  I'll move it to `project.js` as suggested